app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -57,9 +58,13 @@ public DataHandler(Context context) {
         else {
             contactProvider = null;
         }
-
-        if (prefs.getBoolean("enable-search", true)) {
-            providers.add(new SearchProvider(context));
+        switch(prefs.getString("searchEngine", "none")) {
```

Comment:
rather than none, i'd like to use "google".

Otherwise, when we enter a query without going into the settings, we get an empty screen :(